src/pages/home/report/ReportActionsView.js

```diff
@@ -404,6 +435,72 @@ class ReportActionsView extends React.Component {
         }
     }
 
+    /**
+     * Scrolls to a specific report action ID
+     */
+    scrollToReportActionID() {
+        this.actionIndexIDToScroll = _.findIndex(this.sortedReportActions, (
+            ({action}) => parseInt(action.reportActionID, 10) === this.props.reportActionID
+        ));
+
+        if (this.actionIndexIDToScroll !== -1) {
+            ReportScrollManager.scrollToIndex({index: this.actionIndexIDToScroll, viewPosition: 0.5});
+            this.setState({shouldHighlightReportActionID: true});
```

Comment:
Oh because if we pass this.props.reportActionID and not use setState it would highlight the moment the actionID is rendered and not when we scroll to it. Using `shouldHighlightReportActionID` ensures that we're highlighting when we're ready to scroll. Let me know if that makes sense or if you prefer to see this differently.